apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -1,20 +1,224 @@
+    this.checkRating('Accepted')
+  }
+  onReviewApplicationReject = () => {
+    this.checkRating('Rejected')
+  }
+
+  generateRatingChange = () => index => {
+    this.setState({rating: index})
+    console.log('index: ', index)
+  }
+
+  render() {
+    const { issue } = this.props
+    const submittor = work.user
+    const submissionDateDistance = formatDistance(new Date(work.submissionDate), new Date())
+    const ratings = [
+      'Select a Rating',
+      '1 - Unusable', 
+      '2 - Needs Rework', 
+      '3 - Acceptable',
+      '4 - Exceeds Expectations',
+      '5 - Excellent',
+    ]
+
+    return(
+      <div>
+        <IssueTitle>{issue.title}</IssueTitle>
+        
+        <SafeLink
+          href={issue.url}
+          target="_blank"
+          style={{ textDecoration: 'none', color: '#21AAE7' }}
+        >
+          <IssueLinkRow>
+            <IconGitHub color="#21AAE7" width='14px' height='14px' />
+            <Text style={{ marginLeft: '6px'}}>{issue.repo} #{issue.number}</Text>
+          </IssueLinkRow>
+        </SafeLink>
+
+        <SubmissionDetails>
+          <UserLink>
+            <img src={submittor.avatar} style={{ width: '32px', height: '32px', marginRight: '10px'}} />
+            <SafeLink
+              href={submittor.url}
+              target="_blank"
+              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
+            >
+              {submittor.name ? submittor.name : submittor.login}
+            </SafeLink>
+            applied {submissionDateDistance} ago
+          </UserLink>
 
-const ReviewWork = ({ issues, rate, onSubmit }) => <div>Review Work</div>
-
-ReviewWork.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  /** callback fn */
-  onSubmit: PropTypes.func,
+          <Separator/>
+
+          <FieldTitle>Proof of Work</FieldTitle>
+          <SafeLink
+            href={work.proof}
+            target="_blank"
+            style={{ textDecoration: 'none', color: '#21AAE7' }}
+          >
+            <DetailText>{work.proof}</DetailText>
+          </SafeLink>
+
+          <CommentsCheck comments={work.comments} />
+          <DetailText>{work.comments}</DetailText>
+
+          <FieldTitle>Hours Worked</FieldTitle>
+          <DetailText>{work.hours}</DetailText>
+
+        </SubmissionDetails>
+
+        <FormField 
+          label="Quality Rating" 
+          required
+          input={
+            <DropDown
+              items={ratings}
+              onChange={this.generateRatingChange()}
+              active={this.state.rating}
+            />
+          }
+        />
+
+        <FormField
+          label="Feedback"
+          input={
+            <TextInput.Multiline
+              name='feedback'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
```

Comment:
The current multiline from aragon ui has no top-padding should we add a little padding or reuse the [DescriptionInput](https://github.com/AutarkLabs/planning-suite/blob/7daf53218c0af317d7ea3cd645633a525bbf4ea9/apps/allocations/app/components/Form/Input/DescriptionInput.js#L10) component that has the padding already defined? (As seen on Allocations->NewProject)
Or better let it be that way? (no padding Top-bottom) cc @stellarmagnet for this

Reply:
Yes, I noticed that yesterday actually, updated the other panels like this:

https://github.com/AutarkLabs/planning-suite/tree/414-fe-tweaks